src/views/components/createParkingLotForm.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { Button, Container, Alert, Form } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import useCreateParkingLotController from '../../controllers/createParkingLotController.js';
+
+export default function CreateParkingLotForm() {
+  const [name, setName] = useState('');
+  const [capacity, setCapacity] = useState('');
+  const { handleParkingLotCreation, error } = useCreateParkingLotController();
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    handleParkingLotCreation({ name, capacity: parseInt(capacity)});
+  };
+
+  return (
+    <Container className='form-container'>
+      <Form className='form-border' onSubmit={handleSubmit}>
+        <Container className='form-content'>
+
+          <h1 className='form-title'>
+            Create Parking Lot
+          </h1>
+
+          <Form.Group controlId="formBasicLotName">
+            <Form.Label>Name: </Form.Label>
+            <Form.Control 
+              type="text" 
+              placeholder="Enter name" 
+              value={name} 
+              onChange={(e) => setName(e.target.value)} 
+              required 
+              data-test-id="lot-name-input"
+            />
```

Comment:
Let's add `required` tag  - it checks that the input field is not empty when user click submit

Reply:
I believe I have added the required tag there, or is there something else you'd like me to do with it?

Reply:
I am blind, sorry